python/ray/tune/tests/test_tune_restore.py

```diff
@@ -124,11 +124,7 @@ def testAsyncHyperbandExample(self):
 class AutoInitTest(unittest.TestCase):
     def testTuneRestore(self):
         self.assertFalse(ray.is_initialized())
-        tune.run(
-            "__fake",
-            name="TestAutoInit",
-            stop={"training_iteration": 1},
-            ray_auto_init=True)
+        tune.run("__fake", name="TestAutoInit", stop={"training_iteration": 1})
```

Comment:
I think instead we should just make the default name be dated, but allow for override via explicit name